sw/device/lib/dif/dif_kmac_unittest.cc

```diff
@@ -103,6 +103,21 @@ class KmacTest : public testing::Test, public mock_mmio::MmioTest {
       0xa7, 0x48, 0x47, 0x93, 0x0a, 0x03, 0xab, 0xee, 0xa4,
       0x73, 0xe1, 0xf3, 0xdc, 0x30, 0xb8, 0x88, 0x15};
 
+  struct Config_register {
```

Comment:
This might have gotten dropped on the ground, but the name of this struct needs to be `ConfigRegister` to match the style guide. Similarly, the field names should be `foo_bar`.